src/test/java/com/twilio/http/RequestTest.java

```diff
@@ -149,8 +151,8 @@ public void testAddQueryDateTimeRangeClosed() throws MalformedURLException {
     @Test
     public void testAddQueryDateTimeRangeClosedNotUTC() throws MalformedURLException {
         Request r = new Request(HttpMethod.GET, Domains.API.toString(), "/2010-04-01/foobar");
-        r.addQueryDateTimeRange("baz", Range.closed(new DateTime(2014, 1, 10, 14, 0, DateTimeZone.forID("America/Chicago")),
-            new DateTime(2014, 6, 1, 16, 0, DateTimeZone.forID("America/Chicago"))));
+        r.addQueryDateTimeRange("baz", Range.closed(ZonedDateTime.of(2014, 1, 10, 14, 0, 0, 0, ZoneId.of("America/Chicago")),
```

Comment:
I'm having trouble getting this test to pass. It appears that the `ZoneId` is not having any effect. I've tried several zones and get the same result.

Reply:
Add this to _addQueryDateTimeRange_: `.withZoneSameInstant(ZoneOffset.UTC)`